### PR TITLE
DL-13464 Remove wiremock dependency

### PR DESCRIPTION
**Pull request by jmountain-hmrc**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -30,7 +30,6 @@ object AppDependencies {
   )
 
   val itDependencies: Seq[ModuleID] = Seq(
-    "org.wiremock" % "wiremock" % wiremockVersion % Test
```
Comment (nadinJ) [inline project/AppDependencies.scala:33]: wiremockVersion val left in - not used
PR Bot comments not addressed

File: conf/logback-test.xml [hunk 2]
```diff
@@ -14,34 +14,13 @@
...
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="com.google.inject" level="OFF"/>
 
     <logger name="org.asynchttpclient.netty" level="OFF"/>
```
Comment (paulanderson1234) [inline conf/logback-test.xml:26]: These entries relating to netty can also be removed:
logger name="org.asynchttpclient.netty" level="OFF"/
logger name="io.netty.buffer" level="OFF"/
logger name="play.core.netty" level="OFF"

File: conf/logback.xml [hunk 3]
```diff
@@ -35,14 +22,6 @@
...
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="uk.gov" level="INFO"/>
 
     <logger name="org.asynchttpclient.netty" level="INFO"/>
```
Comment (paulanderson1234) [inline conf/logback.xml:27]: This entry can be removed:
logger name="org.asynchttpclient.netty" level="INFO"

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -30,7 +29,6 @@ object AppDependencies {
   )
 
   val itDependencies: Seq[ModuleID] = Seq(
```
Comment (paulanderson1234) [inline project/AppDependencies.scala:31]: probably could close these parenthesis on one line ?

Comment (paulanderson1234) [reply project/AppDependencies.scala]: Refactored

File: conf/logback.xml (hunk 3 above)
Comment (paulanderson1234) [reply conf/logback.xml]: Removed

File: project/AppDependencies.scala (hunk 1 above)
Comment (paulanderson1234) [reply project/AppDependencies.scala:33]: val removed and dependency

**Review by paulanderson1234 — APPROVED** [review]

**Pull request merged**